proto_parser/src/primitive_value_test.cc

```diff
@@ -0,0 +1,132 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#include "primitive_value.h"
+#include "gmock/gmock.h"
+#include "proto/event_predicate.pb.h"
+
+namespace wireless_android_play_analytics{
+ 
+class PrimitiveValueTest : public ::testing::Test {
+ protected:
+  PrimitiveValueTest()
+    : primitive_(PrimitiveValue(/*field_name=*/"test", /*indent_count=*/0, 
+        /*line_number=*/0)) { }
+
+  virtual void SetUp() { }
+
+  PrimitiveValue primitive_;
```

Comment:
You don't need to declare a shared primitive_ here. You can create PrimitiveType in each test.

You also don't need class PrimitiveValueTest here.

Change it from TEST_F to TEST:
TEST(PrimitiveValueTest, ProtoValueGetterTests) {
  PrimitiveValue primitive(/*field_name=*/"test", /*indent_count=*/0, 
        /*line_number=*/0);
  EXPECT_EQ(primitive_.GetName(), "test");
  EXPECT_EQ(primitive_.GetCommentAboveField().size(), 0);
  EXPECT_EQ(primitive_.GetCommentBehindField(), "");
  EXPECT_EQ(primitive_.GetLineNumber(), 0);
  EXPECT_EQ(primitive_.GetIndentCount(), 0);
}

For simple unit tests, no need to have a wrapper class.